addons/mail/static/src/owl/components/moderation_ban_dialog/moderation_ban_dialog.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.component.ModerationBanDialog" owl="1">
+        <div><!-- necessary div for useRef, see https://github.com/odoo/owl/issues/638 -->
+        <Dialog contentClass="'o_ModerationBanDialog'" title="'Confirmation'" size="'medium'" t-ref="dialog">
```

Comment:
untranslated `title`

Reply:
Out of scope, this can't be solved with the current system. See task-2196109